### DISCUSSION TITLE
2023-09-13

### DISCUSSION THREAD

**Discussion by nlom0218** [woowacourse-teams/2023-haru-study#501] [daily_meeting]
### 오늘의 할 일

BE
- DB 스키마
- 인프라 개선하기 과제 업로드

FE
- 오전 : 강의
- 점심 이후 각자 에러 정책 적용

### 개인 일정
노아 5시 근로회의

### 내일의 할 일

**Comment by jaehee329** [comment]
### 체크인
7점 - 일찍 나와서 다행이네요. 근데 우산을 안 들고 온..

### 체크아웃
6점 - 비가 오는데 어떻게 가죠? 정말 걱정스럽습니다 내 노트북이 걱정돼요

**Comment by woosung1223** [comment]
### 체크인
5점 - 30분 낑겨 있었더니 무릎이 아파요

### 체크아웃
5점 - 너무 피곤하지만 그래도 미션을 어느정도 해서 괜찮읍니다

**Comment by MoonJeWoong** [comment]
### 체크인
7점 - 어제 잠은 많이 자서 컨디션은 좋은데 아침부터 잡일이 너무 많아서 7점입니다...

### 체크아웃
8점 - 미션 step1은 어떻게 어떻게 마무리 했네요... 내일도 열심히 해봅시다!

**Comment by woo-jk** [comment]
### 체크인
7점 - 늦어서 땀흘림;;

**Reply by aak2075** [reply]
? 벌써 집에 가시나요

**Comment by nlom0218** [comment]
### 체크인

6점 - 헤모글로빈이 부족한 느낌이에요😭

**Comment by yeopto** [comment]
### 체크인

8점 - 좀 많이 피곤하네요

### 체크아웃

10점 - 에러 처리 끝났다..

**Comment by aak2075** [comment]
### 체크인

10점 - 즐거운 아침입니다
